### ISSUE TITLE
North Atlantic mask misses some ocean area between Greenland and Iceland

### ISSUE THREAD

**Issue by hmkhatri** [jbusecke/cmip6_preprocessing#208]
I followed the approach from [tutorial notebook](https://github.com/jbusecke/cmip6_preprocessing/blob/master/docs/regionmask.ipynb) and created a mask for North Atlantic. However, there is a small region (see while space between Greenland and Iceland in figure) that has nan values, even though there is an ocean in between. I have tried creating masks with 1 deg and 0.25 deg. Both had this issue.

<img width="372" alt="image" src="https://user-images.githubusercontent.com/17830036/155540497-28ffbafb-6114-40bd-ae18-abe522c12740.png">

**Comment by jbusecke** [comment]
Thanks for raising the issue @hmkhatri. The masking is ultimately based on [combining smaller named subregions](https://github.com/jbusecke/cmip6_preprocessing/blob/209041a965984c2dc283dd98188def1dea4c17b3/cmip6_preprocessing/regionmask.py#L7-L136) from [regionmask](https://regionmask.readthedocs.io/en/stable/defined_ocean_basins.html). I suspect that a new one of these regions has been added during a recent [update](https://github.com/regionmask/regionmask/issues/301)

@mathause, is this possible (also sorry for not responding on that old issue)? 

@hmkhatri, could you follow the docs like this:
```
basins = regionmask.defined_regions.natural_earth.ocean_basins_50
basins.plot(add_ocean=False, add_label=False)
```
and zoom in on this region? I would be very curious to see what is going on, but do not currently have time to dig into this myself.

**Comment by mathause** [comment]
Indeed, there is a a bit of ocean not part of any region - but it's the same with both versions of the naturalearthdata. v4.1 is orange and v5.0 is blue.

![ocean_basins_50](https://user-images.githubusercontent.com/10194086/155591867-4b0258fd-bbcf-407e-afb5-13e1edfc764c.png)

@jbusecke you'll be able to specify the naturalearthdata version once the new version is released.

<details>

The code requires regionmask master or the upcoming 0.9 release.
```python

%matplotlib

import cartopy.crs as ccrs
import regionmask

line_kws = dict(color="#ff7f00", lw=2)
ax= regionmask.defined_regions.natural_earth_v4_1_0.ocean_basins_50.plot(resolution="50m", line_kws=line_kws)

line_kws = dict(color="#1f78b4", lw=2, ls=":")
ax = regionmask.defined_regions.natural_earth_v5_0_0.ocean_basins_50.plot(resolution="50m", line_kws=line_kws)
ax.set_extent([-40, -10, 60, 90], ccrs.PlateCarree())
```

</details>

**Comment by hmkhatri** [comment]
@mathause @jbusecke Thanks for looking into this. Seems like some manual tweaking is needed to incorporate the missing region in the North Atlantic mask. 

It is not a major issue and I managed to get rid of white space by combining North Atlantic mask and ocean wet point mask. It would be nice to update the code for correcting it.  I am busy for next few weeks, but I would be happy to work on this later.

**Comment by jbusecke** [comment]
Hey @hmkhatri, does the problem occur if you use
```python
basins = regionmask.defined_regions.natural_earth_v4_1_0
```
as mentioned [here](https://regionmask.readthedocs.io/en/stable/defined_ocean_basins.html)? I have thought about making the basin masking even easier in #119. If the above fixes your problem, I would be inclined to lock down v4.1.0 from natural earth and exclusively use that in this functionality. From my perspective we mostly care about large ocean basins, and for smaller stuff folks could just use regionmaks with their shapefile of choice?

**Comment by jbusecke** [comment]
Bummer to see that this is in fact baked into both NE versions. I have been thinking about refactoring the basin masking ability (#119) and maybe this should be part of it. Using the wetmask is not ideal IMO, since this method should really just rely on the lon/lat values of the model (which makes it quite universal). All that cmip6_pp does is just combine a bunch of shapefiles, so its concievable that we just add a manual correction there? 

Then again, since I wrote this, I have actually used this in many more use cases outside of cmip6. I wonder if such a refactor should maybe include efforts upstream, to just publish a NE shapefile with ocean basins? This seems of much broader use than just for the cmip community. @mathause have you interacted with that community at all?

**Comment by mathause** [comment]
> @mathause have you interacted with that community at all?

No I didn't so far, but you have ;-) nvkelso/natural-earth-vector#319

**Comment by jbusecke** [comment]
I pinged them over in https://github.com/nvkelso/natural-earth-vector/issues/697. Lets see what they think.
